Sources/Transaction/Memos/SenderMemo.swift

```diff
@@ -24,10 +23,11 @@ struct RecoverableSenderMemo {
     }
 
     func recover(senderPublicAddress: PublicAddress) -> SenderMemo? {
-        guard SenderMemoUtils.isValid(memoData: memoData,
-                                   senderPublicAddress: senderPublicAddress,
-                                   receipientViewPrivateKey: accountKey.subaddressViewPrivateKey,
-                                   txOutPublicKey: txOutPublicKey)
+        guard SenderMemoUtils.isValid(
+            memoData: memoData,
+            senderPublicAddress: senderPublicAddress,
+            receipientViewPrivateKey: accountKey.subaddressViewPrivateKey,
+            txOutPublicKey: txOutPublicKey)
```

Comment:
double-indent function call params:
```suggestion
                memoData: memoData,
                senderPublicAddress: senderPublicAddress,
                receipientViewPrivateKey: accountKey.subaddressViewPrivateKey,
                txOutPublicKey: txOutPublicKey)
```

Reply:
single indentation is the general rule